doc/whats_new/v0.20.rst

```diff
@@ -100,3 +100,19 @@ Linear, kernelized and related models
 - Deprecate ``random_state`` parameter in :class:`svm.OneClassSVM` as the
   underlying implementation is not random.
   :issue:`9497` by :user:`Albert Thomas <albertcthomas>`.
+
+Outlier Detection models
+
+- Add a ``score_samples`` method in :class:`svm.OneClassSVM`,
+  :class:`ensemble.IsolationForest`, :class:`neighbors.LocalOutlierFactor`,
+  :class:`covariance.EllipticEnvelope`. It allows to  access raw score
+  functions from original papers. A new ``offset_`` parameter allows to link
+  ``score_samples`` and ``decision_function`` methods.
+  :class:`ensemble.IsolationForest` and :class:`neighbors.LocalOutlierFactor`
+  ``decision_function`` methods now take a ``contamination`` parameter, used
```

Comment:
I'm not sure we should mention ``decision_function`` for LocalOutlierFactor in what's new at it is private. Nothing public has really changed for LocalOutlierFactor except the addition of ``offset_``

Reply:
Still it's the decision function method, even if it is private. I'm not for adding a specific item for LOF, as the changes are basically the same as the others.